core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -142,11 +158,22 @@ public AuthenticationToken getToken() {
       }
     }
 
-    if (securePassword != null)
-      return new PasswordToken(securePassword.value);
-
-    if (password != null)
-      return new PasswordToken(password.value);
+    // other token types should have resolved by this point, so return PasswordToken
+    Password pass = null;
+    if (securePassword != null) {
+      pass = securePassword;
+    } else if (password != null) {
+      pass = password;
+    } else {
+      try {
+        pass = Password.promptUser();
+      } catch (IOException e) {
+        throw new RuntimeException(e);
+      }
+    }
+    if (pass != null) {
```

Comment:
I had something like that before running tests. [TestClientOpts.test ](https://github.com/apache/accumulo/blob/master/core/src/test/java/org/apache/accumulo/core/cli/TestClientOpts.java#L69) expects for that method and ones that call it to return null.